### PR TITLE
add celo chain tokens to prices

### PR DESCRIPTION
**Pull request by RantumBits**
add celo chain tokens to prices

### CODE REVIEW

File: models/prices/celo/prices_celo_tokens.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    token_id
+    , blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+
+    ('celo-celo', 'celo', 'CELO', 0x471ece3750da237f93b8e339c536989b8978a438, 24),
```
Comment (Hosuke) [inline models/prices/celo/prices_celo_tokens.sql:19]: API: https://api.coinpaprika.com/v1/coins/celo-celo
SCAN: https://explorer.celo.org/mainnet/token/0x471EcE3750Da237f93B8E339c536989b8978a438/token-transfers

But the explorer shows its decimal is 18.
```suggestion
    ('celo-celo', 'celo', 'CELO', 0x471ece3750da237f93b8e339c536989b8978a438, 18), 
```

File: models/prices/celo/prices_celo_tokens.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    , blockchain
+    , symbol
+    , contract_address
+    , decimals
+FROM
+(
+    VALUES
+
+    ('celo-celo', 'celo', 'CELO', 0x471ece3750da237f93b8e339c536989b8978a438, 24), 
+    ('cusd-celo-dollar', 'celo', 'CUSD', 0x765de816845861e75a25fca122bb6898b8b1282a, 18)
```
Comment (Hosuke) [inline models/prices/celo/prices_celo_tokens.sql:20]: API: https://api.coinpaprika.com/v1/coins/cusd-celo-dollar
SCAN: https://explorer.celo.org/mainnet/token/0x765de816845861e75a25fca122bb6898b8b1282a/token-transfers

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @RantumBits 

There is a mismatch in celo token though, please verify.

File: models/prices/celo/prices_celo_tokens.sql (hunk 1 above)
Comment (RantumBits) [reply models/prices/celo/prices_celo_tokens.sql:19]: yes you are correct. good catch.

**Review by jeff-dude — APPROVED** [review]
i went ahead and pushed a commit to attempt to finalize:
- add to cross-chain spell which feeds `prices.usd`
- replace legacy logic with dummy model -- since we're fully migrated in prices lineage, no need to recreate there. let's add the dummy just to ensure the schema definition exists

**Pull request merged**